DisplayCAL/lib/agw/artmanager.py

```diff
@@ -503,56 +505,57 @@ def DrawButton(
         dc: wx.DC,
         rect: wx.Rect,
         state: int,
-        input: None | bool | wx.Colour = None,
+        input_: Union[None, bool, wx.Colour] = None,
     ) -> None:
         """Draw a button using the MS Office 2007 theme.
 
         Args:
-            dc (wx.DC): an instance of :class:`wx.DC`.
-            rect (wx.Rect): the button's client rectangle.
-            state (int): the button state.
-            input (None | bool | wx.Colour): a flag used to call the right method.
+            dc (wx.DC): A :class:`wx.DC` instance.
+            rect (wx.Rect): The button's client rectangle.
+            state (int): The button state.
+            input_ (Union[None, bool, wx.Colour]): A flag used to call the right method.
         """
-        if input is None or isinstance(input, bool):
-            self.DrawButtonTheme(dc, rect, state, input)
+        if input_ is None or isinstance(input_, bool):
+            self.DrawButtonTheme(dc, rect, state, input_)
         else:
-            self.DrawButtonColour(dc, rect, state, input)
+            self.DrawButtonColour(dc, rect, state, input_)
 
     def DrawButtonTheme(
-        self, dc: wx.DC, rect: wx.Rect, state: int, useLightColours: None | bool
+        self, dc: wx.DC, rect: wx.Rect, state: int, useLightColours: Union[None | bool]
```

Comment:
Yes this should have been `Union[None, bool]` that's my fault. But you shouldn't use `|` in typehintings as it is introduced in Python 3.10 and we currently support Python 3.8+.

Reply:
That is because the `|` should be a `,`. If you don't use Union, you also drop support for python 3.8 and 3.9. Again, like in my comment, could be the right time to do that. But I think it is important that one is aware of that.